qiskit/transpiler/passes/optimization/commutative_inverse_cancellation.py

```diff
@@ -71,10 +108,14 @@ def run(self, dag: DAGCircuit):
                     not self._skip_node(topo_sorted_nodes[idx2])
                     and topo_sorted_nodes[idx2].qargs == topo_sorted_nodes[idx1].qargs
                     and topo_sorted_nodes[idx2].cargs == topo_sorted_nodes[idx1].cargs
-                    and topo_sorted_nodes[idx2].op == topo_sorted_nodes[idx1].op.inverse()
                 ):
-                    matched_idx2 = idx2
-                    break
+                    result, phase = self._inverse_upto_phase(
```

Comment:
Would `is_equal` be a more concise variable name than `result`? I wasn't sure what I expected this to be until I read the new function added above 🙂

Reply:
Thanks, renamed in [64c934e](https://github.com/Qiskit/qiskit/pull/11248/commits/64c934e692e721e3e3819d5be391707452c7a9ba).